Automattic/wp-calypso
# Add/custom rebrand cities thank you page

This creates a custom thank you page when a user purchases a business plan for a rebrand cities generate site name.

To test:

* run through the signup flow from`/start/rebrand-cities`
* purchase a business plan

You should see the following thank you page for rebrand cities:

<img width="533" alt="screen shot 2017-08-10 at 9 12 04 pm" src="https://user-images.githubusercontent.com/1926/29168088-bc1fd7b6-7e10-11e7-99b5-834ae013cced.png">
